composer.json

```
@@ -4,8 +4,9 @@
     "type": "silverstripe-vendormodule",
     "require": {
         "php": "^8.1",
-        "silverstripe/cms": "^5",
-        "silverstripe/versioned": "^2"
+        "silverstripe/framework": "^5.2",
+        "silverstripe/cms": "^5.2",
+        "silverstripe/versioned": "^2.2"
```

```suggestion
        "silverstripe/cms": "^5",
        "silverstripe/versioned": "^2"
```
No need to bump these.

---

Updated